server/index.js

```diff
@@ -8,16 +8,23 @@ const axios = require('axios');
 const express = require('express');
 const bodyParser = require('body-parser');
 const Overview = require('../database_mongo/index.js');
-const { steam } = require('../database_mongo/iconURLs.js');
 // const data = require('../database_mongo/seed.js');
 
 const app = express();
 
-app.use(express.static('public'));
+app.use(express.static('public', { fallthrough: true }));
+app.use('/:product_id', express.static('public'));
```

Comment:
I was just wondering why there are two middleware that serves the static files

Reply:
aha
I just figured that out today, so my service will fully operate independent of proxy, like you can put in an id after the port and it works, or else defaults to 21

Reply:
AHHHH! That is the coolest discovery by Chris! You've got to hear him out on this MIcko.